LearningMissionSimulation/PlaygroundEdvard.cs

```diff
@@ -252,7 +252,45 @@ public void AssignInstructorsToClassrooms()
 
         public void RegisterStudentsForClasses()
         {
-            throw new NotImplementedException();
+            if (ClassroomDictionary.Count == 0)
```

Comment:
Please add and use:
string action = "Register";
Do the same in all other methods.

Reply:
Resolved.